### DISCUSSION TITLE
Nur eine Klasse als Ergänzung bei Icons

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1085] [Q&A]
Es geht um das Thema svg-Icons farblich anzupassen.
Da einige Icon als iconify-icons nicht zur Verfügung stehen, möchte man svg-Icons nutzen. Diese können aber nicht in ihrer Farbe angepasst werden, so wie bie iconify-Icons.
Daher war meine Idee, ein SVG-Icon als Klasse zu definieren und dann die Farbe in einer weiteren Klasse mit Filter zu verändern.
Beispiel:
.charge {
	color: transparent !important;
	background-image: url("http://192.168.178.89/download/icons/batt_100.svg");
	filter: invert(8%) sepia(100%) saturate(7001%) hue-rotate(248deg) brightness(98%) contrast(145%);
}

.charge .blue {
	background-color: red;
}
Mit der background-color wollte ich nur testen, ob die Klasse überhaupt zieht.
Ist aber leider nicht so. 
Im Icon in jarvis ist eingestellt:
`{"true":"motion-sensor charge blue","false":"motion-sensor-off"}`

Wird in den Icons nur die erste Klasse genutzt?
Wie kann man SVG-Icons farblich ändern, ohne jedesmal das Icon als base64 neu konvertieren zu müssen?

**Comment by mcuiobroker** [comment]
Werden Mehrfach Klassen (s.o.) in jarvis verarbeitet, oder nach der ersten Klassse (charge) beendet?

**Reply by Zefau** [reply]
Sollten alle übernommen werden
